web/src/people.ts

```diff
@@ -1592,7 +1592,18 @@ export function matches_user_settings_search(person: User, value: string): boole
     return safe_lower(person.full_name).includes(value) || safe_lower(email).includes(value);
 }
 
+function filter_for_user_roles(persons: User[], codeString: string): User[] {
```

Comment:
We use snake_case for variable names.

Reply:
not relevant anymore since the code has changed